mission_control/navigator_missions/nodes/move_command

```diff
@@ -22,7 +22,7 @@ def main(args):
 
     action_kwargs = {'move_type': args.movetype, 'speed_factor': args.speedfactor}
 
-    if args.blind is not None:
+    if args.blind not in  (None, False):
```

Comment:
That was the previous behaviour which crashed navc, when it would get passed as a kwarg set to false even though the flag wasn't set on the command line. This was my first fix but I guess I can remove it now.